crates/ev-prover/src/prover/programs/message.rs

```diff
@@ -323,7 +305,7 @@ impl HyperlaneMessageProver {
                 return Err(anyhow::anyhow!("Failed to relay Hyperlane message to Celestia"));
             }
         }
-        info!("[Done] Tia was bridged back to Celestia");
+        info!("[Success] Token was bridged back to Celestia");
```

Comment:
I would suggest that we remove the square bracket notation here around "Success"

I think we could log more useful info. Since we are also potentially proving multiple msgs as we loop above, I think it would be good to log inside the log something like this for example:

```
info!("Successfully submitted process message tx for: {}", message.id);
```

Also as a side note, in future we could bundle all `MsgProcessMessage(s)` into a single tx reducing the number of base fees into a single tx.